.github/workflows/build-html.yml

```diff
@@ -0,0 +1,31 @@
+name: Build Manual
+
+on:
+  push:
+
+jobs:
+  build-html:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: main
+        fetch-depth: 0
+    - name: Setup Graphviz
+      run: sudo apt update && sudo apt install -y graphviz
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+    - name: Install Python dependencies
+      run: pip install --upgrade -r requirements.txt
+    - name: Build versioned HTML manual
+      run: sh build_html.sh
+    - name: Publish
+      uses: netlify/actions/cli@master
+      timeout-minutes: 15
+      with:
+        args: deploy --prod --dir=build/html
+      env:
+        NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
+        NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
```

Comment:
Someone with more permission that I have needs to add these to the Secrets panel in this repository's settings, then we can merge.